maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/api/AssessmentApiServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.laa.crime.orchestration.service.api;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import uk.gov.justice.laa.crime.common.model.ioj.ApiCreateIojAppealRequest;
+import uk.gov.justice.laa.crime.common.model.ioj.ApiCreateIojAppealResponse;
+import uk.gov.justice.laa.crime.common.model.ioj.ApiGetIojAppealResponse;
+import uk.gov.justice.laa.crime.orchestration.client.CrimeAssessmentApiClient;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+@ExtendWith({MockitoExtension.class})
+class AssessmentApiServiceTest {
+    private static final int EXISTING_APPEAL_ID = 1;
+
+    @Mock
+    private CrimeAssessmentApiClient assessmentApiClient;
+
+    @InjectMocks
+    private AssessmentApiService assessmentApiService;
+
+    @Test
+    void givenAppealId_whenFindIsInvoked_thenApiClientReturnsResponse() {
+        when(assessmentApiClient.getIojAppeal(EXISTING_APPEAL_ID)).thenReturn(new ApiGetIojAppealResponse());
```

Comment:
Can remove this stub and the one from `givenCreateIojAppealRequest_whenCreateIsInvoked_thenApiClientReturnsResponse`